Docs/install/install_host.rst

```diff
@@ -94,55 +113,51 @@ Install GPU packages for PyTorch 2.1 or PyTorch 1.13 or ONNX or TensorFlow
     dpkg -i cuda-keyring_1.0-1_all.deb
     apt-get update
 
-Install AIMET packages
-~~~~~~~~~~~~~~~~~~~~~~~
 
-From PyPI
-=========
+Installing AIMET
+================
 
-The default AIMET Torch GPU variant may be installed from PyPI as follows:
-    - Go to https://pypi.org/project/aimet-torch
-    - Browse the Requirements section of each Release to identify the version you wish to install. Following are some tips:
-        - For Pytorch 2.2.2 GPU with CUDA 12.1, use aimet-torch>=1.32.2
-        - For Pytorch 2.1.2 GPU with CUDA 12.1, use aimet-torch==1.32.1.post1
-        - For PyTorch 1.13 GPU with CUDA 11.7, use aimet-torch==1.31.2
+**Choose your AIMET variant.**
 
-Run the following commands to install the package (prepend with "sudo" and/or package version as needed):
+Based on your machine learning framework and GPU preference, choose one of the install procedures below.
 
-.. code-block:: bash
+:ref:`1. Installing AIMET for PyTorch <man-install-torch>`
+
+:ref:`2. Installing AIMET for TensorFlow <man-install-tf>`
+
+:ref:`3. Installing AIMET for ONNX <man-install-onnx>`
+
+.. _man-install-torch:
 
-    apt-get install liblapacke -y
-    python3 -m pip install aimet-torch
+1. Installing AIMET for PyTorch
+-------------------------------
 
-From Release Package
-====================
+**1.1 Select the release tag for the version you want to install, for example, "1.34.0". Releases are listed at:**
```

Comment:
I feel this sentence is too long to be a bold-face section subtitle:
1.1 Select the release tag for the version you want to install, for example, “1.34.0”. Releases are listed at:

Reply:
Modified.